tests/utilities/niswitch_measurement/NISwitchMeasurement.serviceconfig

```diff
@@ -0,0 +1,19 @@
+{
+  "services": [
+    {
+      "displayName": "NI-Switch Measurement (Py)",
```

Comment:
The correct capitalization is "NI-SWITCH": https://www.ni.com/en/support/documentation/release-notes/product.ni-switch.html